dashbuilder-backend/dashbuilder-dataset-cdi/src/main/java/org/dashbuilder/Bootstrap.java

```diff
@@ -0,0 +1,77 @@
+/**
+ * Copyright (C) 2015 JBoss Inc
```

Comment:
The header should not contain references to JBoss Inc (it does not exist for some time now), see https://github.com/droolsjbpm/droolsjbpm-build-bootstrap/blob/master/ide-configuration/LICENSE-ASL-2.0-HEADER.txt